chef/cookbooks/ceph/providers/client.rb

```diff
@@ -60,13 +60,21 @@ def get_caps(ceph_conf, admin_keyring, keyname)
 
 def auth_set_key(ceph_conf, admin_keyring, keyname, caps)
   # try to add the key
-  cmd = "ceph -k #{admin_keyring} -c #{ceph_conf} auth get-or-create #{keyname} #{caps}"
+  caps_tmp = ''
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `caps_tmp`. Did you mean `caps`?<br>Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.